src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,43 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int supply = 0;
+        int buy = 0;
+        int result = 0;
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String line = reader.readLine();
+            while (line != null) {
+                String[] split = line.split(",");
+                String operation = split[0];
+                int number = Integer.parseInt(split[1]);
+                if (operation.equals("supply")) {
+                    supply += number;
+                }
+                if (operation.equals("buy")) {
+                    buy += number;
+                }
+                line = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write out data from a file",e);
```

Comment:
The exception message here is misleading. The operation being performed is reading from a file, not writing. Consider changing the message to 'Can't read data from a file'.